app/src/main/res/values/dimens.xml

```diff
@@ -24,4 +24,8 @@
     <dimen name="sp_18">18sp</dimen>
     <dimen name="dp_200">200dp</dimen>
     <dimen name="dp_40">40dp</dimen>
+    <dimen name="dp_24">24dp</dimen>
```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.dp_24` appears to be unused
 Unused resources make applications larger and slow down builds.